src/peer_manager.rs

```diff
@@ -32,6 +31,11 @@ use std::{
 
 /// Time (in seconds) after which a joining node will get dropped from the map of joining nodes.
 const JOINING_NODE_TIMEOUT_SECS: u64 = 900;
+/// Duration (in seconds) after which a candidate is considered as expired.
+/// Using a larger timeout to allow Online to accumulate via gossip than the prev timeout for
```

Comment:
I think you can remove the part about ` than the prev timeout for grp-msg accumulation.`

Reply:
Not fixed?